src/dmd/func.d

```diff
@@ -2944,9 +2950,9 @@ FuncDeclaration resolveFuncCall(const ref Loc loc, Scope* sc, Dsymbol s,
             printCandidates(loc, td, sc.isDeprecated());
             return null;
         }
-        /* This case happens when several ctors are mixed in an agregate.
+        /* This case happend when several ctors are mixed in an agregate.
```

Comment:
?

Reply:
Hard to say if this still happens.

Reply:
Seems like it's never hit by the test suite:

https://codecov.io/gh/dlang/dmd/compare/fe263f5991ea571b0384c6facfe207a7b17efb44...c3f90f86d8156d2fcb8d10e48add707fb6e6072e/diff

Reply:
I  thought to put `fatal()` instead of returning null actually. If by accident someone manages to trigger the new explicit ICE then there's something to fix anyway.